src/bin/edit/localization.rs

```diff
@@ -439,6 +440,20 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
         /* zh_hans */ "替换",
         /* zh_hant */ "取代",
     ],
+    // EditSelectAll
+[
+/* en */ "Select All",
+/* de */ "Alle auswählen",
+/* es */ "Seleccionar todo",
+/* fr */ "Tout sélectionner",
+/* it */ "Seleziona tutto",
+/* ja */ "すべて選択",
+/* ko */ "모두 선택",
+/* pt_br */ "Selecionar todos",
+/* ru */ "Выбрать всё",
+/* zh_hans */ "全选",
+/* zh_hant */ "全選",
+],
```

Comment:
You need to fix the indentation yourself as rustfmt is disabled in this section. Also, 3 of the translation keys appear to have had subtle mistakes (the German one for sure, as that's my native language). Also, the language comments should be aligned. Like this:
```suggestion
    // EditSelectAll
    [
        /* en      */ "Select All",
        /* de      */ "Alles auswählen",
        /* es      */ "Seleccionar todo",
        /* fr      */ "Tout sélectionner",
        /* it      */ "Seleziona tutto",
        /* ja      */ "すべて選択",
        /* ko      */ "모두 선택",
        /* pt_br   */ "Selecionar tudo",
        /* ru      */ "Выделить всё",
        /* zh_hans */ "全选",
        /* zh_hant */ "全選"
    ]
```